composer/trainer/trainer.py

```diff
@@ -1675,32 +1672,32 @@ def _train_batch(self, use_grad_scaling: bool):
         # Retry until we successfully complete training and return loss
         while True:
             start_time = time.time()
-            total_loss = None
+            total_loss_dict = {'loss/train/total': self._device.tensor_to_device(torch.zeros(size=(1,)))}
             # Note: We use uint8 instead of bool as BOR is not supported on all torch.distributed backends
             should_handle_cuda_oom = 0
             try:
                 assert self.state.scaler is not None
                 microbatches = self._train_data_spec.split_batch(device_batch, self.state.grad_accum)
-                if self.deepspeed_enabled:
-                    total_loss = self._train_microbatches(microbatches)
```

Comment:
Does removing this and adding the if statement in `else:` work? To me, this makes this section a lot cleaner...

Reply:
CC: @jbloxham  for deepspeed question